docs/release/release-tasks.md

```diff
@@ -7,13 +7,13 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Prepare main branch for development of the new release](#prepare-main-branch-for-development-of-the-new-release)
```

Comment:
This seems to diverge from main with this change now (not saying it has to be sync, but not sure about this change)

Reply:
(although having the whole file sync probably makes cherry-picks easier)

Reply:
I'll drop this. We don't update this that much / should have no conflicts.